backend/gradle.properties

```diff
@@ -8,7 +8,8 @@ app.postgres.password=postgres
 
 app.import.xml=https://www.lbe.bayern.de/engagement-anerkennen/ehrenamtskarte/akzeptanzstellen/app-daten.xml
 app.import.json_1=https://bayerische_eak:
-app.improt.json_2=@freinet-online.de/files/api_bayerische_eak/download.php?get_file=freinet_online_akzeptanzpartner.json
+app.import.json_2=@freinet-online.de/files/api_bayerische_eak/download.php?get_file=freinet_online_akzeptanzpartner.json
+app.geocoding.host=nominatim.maps.tuerantuer.org
```

Comment:
We will need to add basic auth to the geocoding service. We can do this in a separate PR though.

https://github.com/digitalfabrik/ehrenamtskarte/issues/442